### ISSUE TITLE
Drug eruption

### ISSUE THREAD

**Issue by Marwaabdelhakim** [obophenotype/human-phenotype-ontology#11133]
**Preferred term label:**

Drug eruption
**Synonyms**

drug-induced cutaneous reaction
**Definition (free text, please give PubMed ID)**

Acute or subacute adverse cutaneous reactions to a drug or medicine include drug eruptions
**Parent term (use hpo.jax.org/app)**
Adverse drug response 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
[0000-0001-6816-2119](https://orcid.org/0000-0001-6816-2119)

**Comment by pnrobinson** [comment]
@Marwaabdelhakim 
This is not really a phenotypic observation, but it refers to a large number of different phenotypes caused as a reaction to a drug: https://www.ncbi.nlm.nih.gov/books/NBK533000/
This is not in the scope of the HPO, we would code the observed phenotypic abnormalities.
